draft-ietf-quic-http.md

```diff
@@ -884,7 +884,8 @@ as-yet-unknown Push ID, both the associated client request and the pushed
 request header fields are unknown.  The client can buffer the stream data in
 expectation of the matching PUSH_PROMISE. The client can use stream flow control
 (see {{Section 4.1 of QUIC-TRANSPORT}}) to limit the amount of data a server may
-commit to the pushed stream.
+commit to the pushed stream.  Clients SHOULD discard data from push streams if
```

Comment:
Does the client also abort reading the push stream if it is incomplete?

Reply:
If what is incomplete?  The Push ID?  The response?

Reply:
I think the PR is clear that a client needs to process any push stream at least as far as the push ID.  

The new text says the client should 'discard data' if the matching PUSH_PROMISE doesn't arrive in a reasonable time.  Should the client also send a STOP_SENDING in this case?  By incomplete I meant 'had received the Push ID but not the FIN'

Reply:
Ah, I see.  Thanks for clarifying!  I think 0ce850eb addresses this better.